pandas/core/generic.py

```diff
@@ -12,6 +12,7 @@
 from typing import (
     TYPE_CHECKING,
     Any,
+    AnyStr,
```

Comment:
I assume something went wrong here?

Reply:
We didn't really touch generic.py, we are confused as to what is going wrong here.  First time working with this library, so any context would be appreciated.

Reply:
To add to this, we followed the contributing guidelines instructions about merging upstream with our feature branch and it appears to have introduced the AnyStr addition to generic.py

Reply:
Looks like your latest merge fixed it